GAME/GAMEFLOW.C

```diff
@@ -201,6 +202,8 @@ void LoadGameflow()//102E0, 102B0
 	FILE_Load(GF_SCRIPT_FILENAME, s);
 #endif
 
+	gfScriptFile = s;
```

Comment:
Why? IIRC this is not done on PSX. PC could be different but maybe line 207 is what you want (same thing).